client/my-sites/upgrades/checkout/domain-details-form.jsx

```diff
@@ -241,7 +296,6 @@ export default React.createClass( {
 
 	handlePrivacyDialogSelect( options ) {
 		this.formStateController.handleSubmit( ( hasErrors ) => {
-			this.setState( { isDialogVisible: false } );
```

Comment:
I had to remove this line because a warning was being thrown when we proceeded to the payment screen (thus unmounting this dialog) while the dialog was being closed. This is related to https://github.com/facebook/react/issues/4876